ui/src/app/views/app/component.ts

```diff
@@ -24,6 +24,7 @@ export class AppComponent implements OnInit {
   private baseTitle: string;
   user: Observable<SignInDetails>;
   hasAdminPermissions = false;
+  private _showCreateAccount = false;
```

Comment:
For my understanding, is there a particular reason for the underscore convention?

Reply:
It otherwise conflicts with the function of the same name. Naming suggestions welcome.

Reply:
I don't have a better name for it, and I think the underscore is fine. I almost prefer the mirrored names to trying to remove the underscore and leaving it less coupled.